components/inflector.rst

```diff
@@ -60,5 +60,5 @@ forms::
     Inflector::singularize('indices'); // ['index', 'indix', 'indice']
     Inflector::singularize('leaves');  // ['leaf', 'leave', 'leaff']
 
-    Inflector::pluralize('matrix');    // ['matricies', 'matrixes']
+    Inflector::pluralize('matrix');    // ['matrices', 'matrixes']
```

Comment:
If this is correct we should check our code of the pluralize method too, maybe we also did this mistake there 😉

Reply:
https://github.com/symfony/symfony/pull/44632